jupyter_server/extension/application.py

```diff
@@ -284,8 +274,10 @@ def initialize_templates(self):
 
     def _prepare_config(self):
         """Builds a Config object from the extension's traits and passes
-        the object to the webapp's settings as `<extension_name>_config`.  
+        the object to the webapp's settings as `<extension_name>_config`.
         """
+        # Verify all traits are up-to-date with config
```

Comment:
I wish this comment was more about *why* instead of *what*. The comment also seems somewhat at odds with the method name (in which direction does the information flow, from traits -> config, or the other way around?).

Reply:
The lines above were removed. they weren't need after all.